vm/systemSmartContracts/delegation.go

```diff
@@ -602,19 +630,24 @@ func (d *delegation) stakeNodes(args *vmcommon.ContractCallInput) vmcommon.Retur
 		return vmcommon.UserError
 	}
 
+	//TODO: Missing this line, before calling saveGlobalFundData?: globalFund.TotalActive.Sub(globalFund.TotalActive, stakeValue)
```

Comment:
no need to do the sub. saveGlobalFundData can be deleted. The previous implementation with sub from totalactive was wrong.

Reply:
Ok, deleted